code/modules/jobs/job_types/atmospheric_technician.dm

```diff
@@ -9,6 +9,8 @@
 	supervisors = "the chief engineer"
 	selection_color = "#fff5cc"
 	chat_color = "#D4A07D"
+	minimal_player_age = 7
+	exp_requirements = 600
```

Comment:
10 hours of crew playtime to be an atmos tech?

Reply:
Atmos tech is an advanced job, new accounts should not be able to access it right away. Also there is problem with people just flooding station with plasma...

Reply:
I strongly back the belief that all of engineering should be locked behind at least 10 hours of general play.